dev/restinio/chunked_input_info.hpp

```diff
@@ -69,7 +97,7 @@ class chunk_info_t
 	 * is undefined behavior.
 	 */
 	RESTINIO_NODISCARD
-	string_view_t
+	[[nodiscard]] string_view_t
```

Comment:
There is no need to specify `[[nodiscard]]` attribute if RESTINIO_NODISCARD is used because RESTINIO_NODISCARD should expands to `[[nodiscard]]`

Reply:
Since we switch to C++17, it looks reasonable to simply assume `[[nodiscard]]` is available?

Reply:
Yes. I'll replace RESTINO_NODISCARD to simple `[[nodiscard]]` later.